app/models/spree/image.rb

```diff
@@ -5,9 +5,14 @@ class Image < Asset
     validates_attachment_presence :attachment
     validate :no_attachment_errors
 
+    # This is where the styles are used in the app:
+    # - mini: used in the BackOffice: Bulk Product edit page and Order Cycle edit page
+    # - small: used in the FrontOffice: product list page
+    # - product: used in the BackOffice: Product Image upload modal in the Bulk Product edit page and Product image edit page
+    # - large: used in the FrontOffice: product modal
     has_attached_file :attachment,
-                      styles: { mini: '48x48>', small: '100x100>',
-                                product: '240x240>', large: '600x600>' },
+                      styles: { mini: ["48x48#", :jpg], small: ["227x227#", :jpg],
+                                product: ["240x240>", :jpg], large: ["600x600>", :jpg] },
```

Comment:
Don't we need to deploy this immediately after we convert the styles with the task?

Reply:
Yes, so #6014 needs to be deployed already. It could avoid delays if we run the scripts in #6014 early then run it again with the new images just before we want to deploy this here.